### ISSUE TITLE
Field with name "class" generates invalid Java code

### ISSUE THREAD

**Issue by kshchepanovskyi** [protostuff/protostuff-compiler#16]
A field named `class` will generate invalid Java code - the corresponding `getClass()` method is already defined in the `Object` superclass.

The following error is reported when compiling:

```
error: getClass() in XXX cannot override getClass() in Object
```

`protoc` generates following java code:

```java
/**
 * <code>optional int32 class = 1;</code>
 */
int getClass_();
```

Source: https://github.com/protostuff/protostuff/issues/163
